thecodingmachine/tdbm-bundle/5.0/config/packages/tdbm.yaml

```diff
@@ -0,0 +1,18 @@
+tdbm:
+    bean_namespace: App\Beans
+    dao_namespace: App\Daos
+
+## Uncomment the naming section if you want to customize the names of the generated classes / beans
+#    naming:
+#        bean_prefix: ""
+#        bean_suffix: ""
+#        base_bean_prefix: Abstract
```

Comment:
We should not have config in comments like this.

I would much rather see a link to the documentation.